socorro/schemas/java_exception.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$target_version": 2,
+  "type": "object",
+  "description": "JSON Schema for the JavaException crash annotation.",
+  "definitions": {},
+  "required": ["exception"],
+  "properties": {
+    "exception": {
+      "type": "object",
+      "required": ["values"],
+      "properties": {
+        "values": {
+          "type": "array",
+          "description": "Array of exception stacktraces",
+          "items": {
+            "type": "object",
+            "required": ["stacktrace"],
+            "properties": {
+              "stacktrace": {
+                "type": "object",
+                "description": "A single stacktrace",
+                "required": ["frames", "type", "module"],
+                "properties": {
+                  "frames": {
+                    "type": "array",
+                    "description": "Array of frames",
+                    "items": {
+                      "type": "object",
+                      "description": "A single frame",
+                      "required": ["module", "function", "in_app", "lineno"],
```

Comment:
I looked at 10,000 JavaException values. Some of them had frames that don't have a filename. I don't know offhand what causes that in Java land. Anyhow, "filename" shouldn't be required here.